lib/libsys/interposing_table.c

```diff
@@ -41,7 +41,10 @@
 int
```

Comment:
Does this comment need adjusting now?  It seems the wrapper will still be needed even with function pointer wrapping?

Reply:
Hmmm, I removed the comment earlier but it came back after a rebase. I'll remove it.